### PR TITLE
CS Ed Week 2021 blog post

### LINKED ISSUE #8579: CS Ed Week blog post
Hey Galen,
Can you prep this blog post to go live on Monday?
Files here - https://1drv.ms/u/s!AmMIW5Hxi0RtgZ88DHJG1Q2HB3GBZA?e=qvFxpK
Thx!
Jacqueline

### PR DESCRIPTION
**Pull request by ganicke**
Import, format, adjust images, and edits for the CS Ed Week 2021 blog post.

Closes #8579

### CODE REVIEW

File: docs/blog/csed-week/csed-week-2021.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+[Computer Science Education Week](www.https://csedweek.org) is coming up December 6th - 12th. This is an annual event to raise the visibility of computer science in education, and to inspire teachers and students to complete an "Hour of Code" during this week.
+
+For those of you already thinking about planning activities for CS Ed Week, here are some fun ideas using MakeCode!
+
+## MakeCode Arcade
+
+The theme for this year’s CS Ed Week is "CS is Everywhere" – highlighting how technology and computer science are helping us solve some of the world’s toughest problems.
+
+The 2021 MakeCode Arcade activity focuses on the impact that computing can have in savigfthe environment – and specifically how computers and technology can help combat forest fires. The **[Save the Forest!](https://arcade.makecode.com/hour-of-code-2021)** activity guides students through creating a game that will start and spread a forest fire, and then code their airtanker plane to spray water to put out the flames.
```
Comment (Jaqster) [inline docs/blog/csed-week/csed-week-2021.md:13]: typo "savigfthe"

File: docs/blog/csed-week/csed-week-2021.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Learn more at: https://arcade.makecode.com/gamejam/global-2021.
+
+## MakeCode in Minecraft
+
+[Minecraft: Education Edition](https://education.minecraft.net/en-us/homepage) is hosting another free coding world for anyone to download and play – you don’t need to buy a license or have an Office 365 account!
+
+![HoC 2021 splash banner](/static/blog/csed-week/csed-week-2021/hoc2021-splash-banner.jpg)
+
+This year’s Hour of Code experience is called "TimeCraft" where your player will travel back in time to fix **Time Splits** (mysterious errors in history) with the help of your Time Agent! There are 10 different coding challenges help save Jazz in the 1920’s, or to help change the shape of the Pyramids in Egypt from squares to triangles.
```
Comment (Jaqster) [inline docs/blog/csed-week/csed-week-2021.md:33]: This reads a bit odd... maybe change to, "There are 10 different coding challenges to fix these time splits.  For example, go back in time to help save Jazz in the 1920's, or help change the shape of the Pyramids in Egypt from squares back to triangles!"

File: docs/blog/csed-week/csed-week-2021.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+![HoC 2021 Minecraft jazz musician](/static/blog/csed-week/csed-week-2021/hoc2021-jazz-musician.png)
+
+Head over to https://education.minecraft.net/en-us/resources/hour-code-2021 to download the game.
+
+## MakeCode for micro:bit
+
+If you have micro:bits in the classroom, there are many fun introductory projects that will light up imaginations and creativity through physical computing. One of the most popular projects is the [micro:pet](https://microbit.org/projects/make-it-code-it/microbit-pet/
```
Comment (Jaqster) [inline docs/blog/csed-week/csed-week-2021.md:41]: fix link

**Review by Jaqster — CHANGES_REQUESTED** [review]
Just a few edits...  Thanks Galen!

File: docs/blog/csed-week/csed-week-2021.md (hunk 2 above)
Comment (ganicke) [reply docs/blog/csed-week/csed-week-2021.md]: How about: "There are 10 different coding challenges to fix these time splits. In one of the challenges, the player goes back in time to help save Jazz in the 1920's. In another, the player helps to change the Pyramids in Egypt from square shapes back into triangles!"

**Comment by ganicke** [comment]
@Jaqster - the Minecraft HoC link page is live now. Want to post?

**Comment by Jaqster** [comment]
Oh, yes please! :-)

**Comment by ganicke** [comment]
Go ahead and tag the **Approval** box for me, thanks.

**Review by Jaqster — APPROVED** [review]

**Pull request merged**